### PR TITLE
Update: Added dropdown to display speakers, schedule and tutorial menu

### PR DESCRIPTION
**Pull request by Amanya-Brian**

### CODE REVIEW

File: components/ui/navbar.js [hunk 1]
```diff
@@ -9,12 +9,22 @@ import { useState } from "react";
...
   const NavbarLinks = [
     {
-      path: "/speakers",
-      label: "Speakers",
-    },
+      path: "#",
+      label: "Program Guide",
+      subLinks: [
+        {
+          path: "/speakers",
```
Comment (HassanBahati) [inline components/ui/navbar.js:16]: last Saturday we had a request to keep `schedule` and `speakers` at top level. am thinking move towards having the `dango girls` under a `community ` dropdown

**Comment by Amanya-Brian** [comment]
Thank you

File: components/ui/navbar.js (hunk 1 above)
Comment (basiima) [reply components/ui/navbar.js:16]: totally agree, speakers and schedule need to remain seen in the navbar

Comment (Amanya-Brian) [reply components/ui/navbar.js]: I have restored the Schedule and Speakers but added Tutorials dropdown to link to Django Girls

**Review by basiima — APPROVED** [review]
this is fine

**Pull request merged**